include/ibamr/IBImplicitStaggeredHierarchyIntegrator.h

```diff
@@ -150,11 +156,113 @@ class IBImplicitStaggeredHierarchyIntegrator : public IBHierarchyIntegrator
 
 private:
     /*!
-     * \brief Default constructor.
-     *
-     * \note This constructor is not implemented and should not be used.
-     */
-    IBImplicitStaggeredHierarchyIntegrator();
+	 * \brief A StaggeredStokesSolver that does nothing.
```

Comment:
> On Apr 27, 2015, at 1:42 PM, Amneet Bhalla <notifications@github.com> wrote:
> 
> In include/ibamr/IBImplicitStaggeredHierarchyIntegrator.h <https://github.com/IBAMR/IBAMR/pull/6#discussion_r29170545>:
> 
> > @@ -150,11 +156,113 @@ class IBImplicitStaggeredHierarchyIntegrator : public IBHierarchyIntegrator
> >  
> >  private:
> >      /*!
> > -     * \brief Default constructor.
> > -     *
> > -     * \note This constructor is not implemented and should not be used.
> > -     */
> > -    IBImplicitStaggeredHierarchyIntegrator();
> > +	 * \brief A StaggeredStokesSolver that does nothing.
> Yeah, the SNES function works with inhomogeneous BC incorporated into Stokes Op, so we don't need to worry about subtracting inhomogeneous BCs from RHS.
> 
This isn't really about boundary conditions; this is about the extra stuff in the IB-Stokes momentum operator (i.e., (rho/dt)*I - (mu/2)*L - (dt/4)*S*A*J). I think that the code is set up so that you can pass in a Newton solver as the Stokes solver. Such a solver can handle inhomogeneous boundary conditions.